mkosi/machine.py

```diff
@@ -122,9 +127,16 @@ def run(self, commands: Sequence[str], timeout: int = 900, check: bool = True) -
 
         return process
 
-    def __exit__(self, *args: Any, **kwargs: Any) -> None:
+    def __kill_image(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
You can name this _kill, no need for the image suffix. Since this is a method of the machine class, it's pretty obvious that the _kill method means that we're killing the machine. Same applies to the _boot_image method.

Reply:
I assume the same for `_reboot_image` then?